Add support for multiple JWT public keys (#327)
neicnordic/sensitive-data-archive: sda/cmd/s3inbox/s3inbox.md

- `INBOX_ACCESSKEY`: The S3 access and secret key are used to authenticate to S3,
 [more info at AWS](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys)
- `INBOX_SECRETKEY`: The S3 access and secret key are used to authenticate to S3,
 [more info at AWS](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys)
- `INBOX_BUCKET`: The S3 bucket to use as the storage root
- `INBOX_PORT`: S3 connection port (default: `443`)
- `INBOX_REGION`: S3 region (default: `us-east-1`)
- `INBOX_CHUNKSIZE`: S3 chunk size for multipart uploads.
- `INBOX_CACERT`: Certificate Authority (CA) certificate for the storage system, this is only needed if the S3 server has a certificate signed by a private entity

### Logging settings

- `LOG_FORMAT` can be set to “json” to get logs in json format. All other values result in text logging
- `LOG_LEVEL` can be set to one of the following, in increasing order of severity:
  - `trace`
  - `debug`
  - `info`
  - `warn` (or `warning`)
  - `error`
  - `fatal`
  - `panic`

## Service Description

The s3inbox proxies uploads to an S3 compatible storage backend.

> 1. Parses and validates the JWT token (`access_token` in the S3 config file) against teh public keys, either local files on sidk or OIDC JWK endpoints.

Reviewer: ```suggestion
1. Parses and validates the JWT token (`access_token` in the S3 config file) against the public keys, either locally provisioned or from OIDC JWK endpoints.
```

typo fix, some fuzziness added.
